packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -267,7 +267,7 @@ async function fetchLatestSnapshotCore(
                     const fluidEpoch = response.odspSnapshotResponse.headers.get("x-fluid-epoch");
                     assert(fluidEpoch !== undefined, 0x1e6 /* "Epoch  should be present in response" */);
                     const valueWithEpoch: IVersionedValueWithEpoch = {
-                        value: snapshot,
+                        value: { snapshot, entryTime: Date.now() },
```

Comment:
This should be {...snapshot, entryTime: Date.now() },

Reply:
BTW, we should be able to detect bugs like this. Does it show up when using any of the examples? If not, this likely points to a fact that we need to add cache capabilities to webpack loader.
Stress tests should find it though, we definitely use caching there.

Reply:
Added type before putting value.